tests/integ_tests/metrics_tests.rs

```diff
@@ -28,7 +28,7 @@ async fn prometheus_metrics_exported() {
     let addr = rt.telemetry().prom_port().unwrap();
     let opts = get_integ_server_options();
     let mut raw_client = opts
-        .connect_no_namespace(rt.metric_meter(), None)
+        .connect_no_namespace(rt.metric_meter().as_deref(), None)
```

Comment:
Note for SDK authors - you'll need to add this small change to your bridge code.